settingscheck.py

```diff
@@ -0,0 +1,35 @@
+import settings
```

Comment:
I thought it made sense to check the structure of the settings in case someone wanting to use the script changes them and messes up

Reply:
instead of using a settings.py file, this project should start using `argparse` which also has type checking.
https://github.com/pablobuenaposada/attendin/blob/main/src/main.py#L39

Reply:
Then we'd need to pass the booking goals via arguments? I can do that. Maybe in another PR?